### ISSUE TITLE
Go install fails with replace directive error

### ISSUE THREAD

**Issue by tkjaer** [josephburnett/jd#78]
`go install` of the latest version fails with a replace directive error:

```
tkjaer@ubuntu:~ $ go install github.com/josephburnett/jd@latest
go: downloading github.com/josephburnett/jd v1.9.0
go: github.com/josephburnett/jd@latest (in github.com/josephburnett/jd@v1.9.0):
    The go.mod file for the module providing named packages contains one or
    more replace directives. It must not contain directives that would cause
    it to be interpreted differently than if it were the main module.
tkjaer@ubuntu:~ $
```

```
tkjaer@ubuntu:~ $ go install github.com/josephburnett/jd@v1.8.1
go: downloading github.com/josephburnett/jd v1.8.1
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/go-openapi/jsonpointer v0.19.5
go: downloading github.com/go-openapi/swag v0.21.1
go: downloading github.com/mailru/easyjson v0.7.7
go: downloading github.com/josharian/intern v1.0.0
tkjaer@ubuntu:~ $
```

**Comment by pskrbasu** [comment]
+1

**Comment by josephburnett** [comment]
Dang! I was trying to import the v2 API into main, but still keep it a separate module. I'll just leave it as part of the same module, it's not the worth the headache. Sorry about that!

I just published v1.9.1 with only that change. I created a fresh VM and ran `go install ...` to verify it works. :+1: 

Let me know if it works for you and we can close this issue. Thank you for the report! :pray:
